fastlane/.xcovignore

```diff
@@ -11,3 +11,6 @@
 - Sources/PaymentsCore/Networking/URLSession+URLSessionProtocol.swift
 # Exclude WebAuthSession for now
 - Sources/PaymentsCore/WebAuthenticationSession.swift
+# Excluding PayPal native checkout until figured out mocking interfaces with oxo
```

Comment:
What is this file for?

Reply:
Ah, I see https://github.com/paypal/iOS-SDK/pull/69/files#r945901294

Reply:
Do we have a follow up ticket to add this back in and test it? Feel a bit odd to leave PayPalClient untested. Also is PayPalClient still the best name for this or do we need to be more specific to the feature?

Reply:
this is also just following android, I just paused development for this PR, because won't be able to complete until I get descision on callbacks, let me know if any suggestions for the name

Reply:
> won't be able to complete until I get descision on callbacks

Can you elaborate on what this means and why these files were added to `xcovignore`?